configuration/configuration.go

```diff
@@ -373,4 +391,8 @@ const (
 
 	defaultWitURL     = "https://api.prod-preview.openshift.io/api/"
 	defaultTogglesURL = "http://f8toggles/api"
+
+	varAuthGrantType = "AUTH_GRANT_TYPE"
```

Comment:
Since these are not config, but just constant name, do a `const` declaration of these in the relevant package.

Reply:
done